airflow/configuration.py

```diff
@@ -458,6 +464,12 @@ def run_command(command):
 scheduler_heartbeat_sec = 5
 authenticate = true
 max_threads = 2
+
+# Turn off scheduler backfills by setting this to false.
+# Default behavior is unchanged
+# Command Line Backfills still work, but the scheduler
+# will not do backfills if this is false.
+backfill_by_default = true
```

Comment:
Yes, that is what it looks like to me as well, first default, then test. However, due to the day job, I didn't track that one down.

I had the same issue with the config.getboolean(), and decided to leave it for now and circle back once our showstoppers where fixed. In addition, I wanted to keep this PR clean and not involve extraneous fixes.